NearbyShare/InboundNearbyConnection.swift

```diff
@@ -329,6 +329,7 @@ class InboundNearbyConnection: NearbyConnection{
 	private func acceptTransfer(){
 		do{
 			for (id, file) in transferredFiles{
+				SleepManager.shared.disableSleep()
```

Comment:
Could you move this outside of the for loop.
also you enableSleep when the connection is terminated, why not enableSleep after the for loop? 
it is quite disconnect to have disable in this file and enable in the other file